src/core/ext/transport/chaotic_good/frame_header.cc

```diff
@@ -60,9 +60,15 @@ absl::StatusOr<FrameHeader> FrameHeader::Parse(const uint8_t* data) {
   header.flags = BitSet<2>::FromInt(flags);
   header.stream_id = ReadLittleEndianUint32(data + 4);
   header.header_length = ReadLittleEndianUint32(data + 8);
+  if (header.flags.is_set(0) && !(header.header_length > 0)) {
```

Comment:
how about:
`if (header.flags.is_set(0) && header.header_length <= 0)`

Reply:
Updated, thanks for the suggestion.